Dockerfile

```diff
@@ -49,6 +50,10 @@ RUN mkdir -p /grpc-java && \
     rm -Rf /grpc-java && \
     rm -Rf /grpc
 
+ARG GRPC_WEB_VERSION
+RUN curl -sSLO https://github.com/grpc/grpc-web/releases/download/${GRPC_WEB_VERSION}/protoc-gen-grpc-web-${GRPC_WEB_VERSION}-linux-x86_64 && \
```

Comment:
@rubens21 we are using a multi-arch build (for amd64 and arm64). I see that at least in v1.5.0 of GPRC_WEB they publish aarch64 build, so if you want to change this to work with both, we could merge.

I would've preferred to have some test. Right now we test the build by running `make proto` in jaeger-idl, which I think exercises multiple languages. We could include the new TS output there.